integration-tests/pkg/assert/assert.go

```diff
@@ -0,0 +1,53 @@
+package assert
+
+import (
+	"encoding/json"
+	"github.com/stackrox/collector/integration-tests/pkg/collector"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stackrox/collector/integration-tests/pkg/types"
+)
+
+func AssertExternalIps(t *testing.T, enable bool, collectorIP string) {
+	AssertRepeated(t, func() bool {
+		body, err := collector.IntrospectionQuery(collectorIP, "/state/config")
```

Comment:
I think it can be done in another PR. Maybe we should have an `Introspection` struct that could be a member of a `Collector` interface. The introspection object would get the IP address from the collector object.